### ISSUE TITLE
Bug: Restart loop after latest update (commit 7d824a5)

### ISSUE THREAD

**Issue by hugalafutro** [qdm12/gluetun#772]
### Is this urgent?

Yes

### Host OS

ubuntu

### CPU arch

x86_64

### VPN service provider

Private Internet Access

### What are you using to run the container

docker-compose

### What is the version of Gluetun

latest built on 2022-01-06T11:44:16.796Z (commit 7d824a5)

### What's the problem 🤔

container doesn't come up. stuck in restart loop

### Share your logs

```log
(I changed few chars in the long-ass string in case it's something personal)

pia_gluetun_vm-docker  | ========================================
pia_gluetun_vm-docker  | ========================================
pia_gluetun_vm-docker  | =============== gluetun ================
pia_gluetun_vm-docker  | ========================================
pia_gluetun_vm-docker  | =========== Made with ❤️ by ============
pia_gluetun_vm-docker  | ======= https://github.com/qdm12 =======
pia_gluetun_vm-docker  | ========================================
pia_gluetun_vm-docker  | ========================================
pia_gluetun_vm-docker  |
pia_gluetun_vm-docker  | Running version latest built on 2022-01-06T11:44:16.796Z (commit 7d824a5)
pia_gluetun_vm-docker  |
pia_gluetun_vm-docker  | 🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
pia_gluetun_vm-docker  | 🐛 Bug? https://github.com/qdm12/gluetun/issues/new
pia_gluetun_vm-docker  | ✨ New feature? https://github.com/qdm12/gluetun/issues/new
pia_gluetun_vm-docker  | ☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
pia_gluetun_vm-docker  | 💻 Email? quentin.mcgaw@gmail.com
pia_gluetun_vm-docker  | 💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
pia_gluetun_vm-docker  | 2022/01/06 14:44:41 INFO storage: merging by most recent 11120 hardcoded servers and 11120 servers read from /gluetun/servers.json
pia_gluetun_vm-docker  | 2022/01/06 14:44:41 ERROR failed validating VPN settings: provider settings validation failed: server selection settings validation failed: OpenVPN server selection settings validation failed: PIA encryption preset is not valid: strong; valid presets are none, normal or MIIHqzCCBZOgAwIBAgIJAJ0u+vODZJntMA0GCSqGSIb3DQEBDQUAMIHoMQswCQYDVQQGEwJVUzELMAkGA1UECBMCQ0ExEzARBgNVBAcTCkxvc0FuZ2VsZXMxIDAeBgNVBAoTF1ByaXZhdGUgSW50ZXJuZXQgQWNjZXNzMSAwHgYDVQQLExdQcml2YXRlIEludGVybmV0IEFjY2VzczEgMB4GA1UEAxMXUHJpdmF0ZSBJbnRlcm5ldCBBY2Nlc3MxIDAeBgNVBCkTF1ByaXZhdGUgSW50ZXJuZXQgQWNjZXNzMS8wLQYJKoZIhvcNAQkBFiBzZWN1cmVAcHJpdmF0ZWludGVybmV0YWNjZXNzLmNvbTAeFw0xNDA0MTcxNzQwMzNaFw0zNDA0MTIxNzQwMzNaMIHoMQswCQYDVQQGEwJVUzELMAkGA1UECBMCQ0ExEzARBgNVBAcTCkxvc0FuZ2VsZXMxIDAeBgNVBAoTF1ByaXZhdGUgSW50ZXJuZXQgQWNjZXNzMSAwHgYDVQQLExdQcml2YXRlIEludGVybmV0IEFjY2VzczEgMB4GA1UEAxMXUHJpdmF0ZSBJbnRlcm5ldCBBY2Nlc3MxIDAeBgNVBCkTF1ByaXZhdGUgSW50ZXJuZXQgQWNjZXNzMS8wLQYJKoZIhvcNAQkBFiBzZWN1cmVAcHJpdmF0ZWludGVybmV0YWNjZXNzLmNvbTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBALVkhjumaqBbL8aSgj6xbX1QPTfTd1qHsAZd2B97m8Vw31c/2yQgZNf5qZY0+jOIHULNDe4R9TIvyBEbvnAg/OkPw8n/+ScgYOeH876VUXzjLDBnDb8DLr/+w9oVsuDeFJ9KV2UFM1OYX0SnkHnrYAN2QLF98ESK4NCSU01h5zkcgmQ+qKSfA9Ny0/UpsKPBFqsQ25NvjDWFhCpeqCHKUJ4Be27CDbSl7lAkBuHMPHJs8f8xPgAbHRXZOxVCpayZ2SNDfCwsnGWpWFoMGvdMbygngCn6jA/W1VSFOlRlfLuuGe7QFfDwA0jaLCxuWt/BgZylp7tAzYKR8lnWmtUCPm4+BtjyVDYtDCiGBD9Z4P13RFWvJHw5aapx/5W/CuvVyI7pKwvc2IT+KPxCUhH1XI8ca5RN3C9NoPJJf6qpg4g0rJH3aaWkoMRrYvQ+5PXXYUzjtRHImghRGd/ydERYoAZXuGSbPkm9Y/p2X8unLcW+F0xpJD98+ZI+tzSsI99Zs5wijSUGYr9/j18KHFTMQ8n+1jauc5bCCegN27dPeKXNSZ5riXFL2XX6BkY68y58UaNzmeGMiUL9BOV1iV+PMb7B7PYs7oFLjAhh0EdyvfHkrh/ZV9BEhtFa7yXp8XR0J6vz1YV9R6DYJmLjOEbhU8N0gc3tZm4Qz39lIIG6w3FDAgMBAAGjggFUMIIBUDAdBgNVHQ4EFgQUrsRtyWJftjpdRM0+925Y6Cl08SUwggEfBgNVHSMEggEWMIIBEoAUrsRtyWJftjpdRM0+925Y6Cl08SWhge6kgeswgegxCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTETMBEGA1UEBxMKTG9zQW5nZWxlczEgMB4GA1UEChMXUHJpdmF0ZSBJbnRlcm5ldCBBY2Nlc3MxIDAeBgNVBAsTF1ByaXZhdGUgSW50ZXJuZXQgQWNjZXNzMSAwHgYDVQQDExdQcml2YXRlIEludGVybmV0IEFjY2VzczEgMB4GA1UEKRMXUHJpdmF0ZSBJbnRlcm5ldCBBY2Nlc3MxLzAtBgkqhkiG9w0BCQEWIHNlY3VyZUBwcml2YXRlaW50ZXJuZXRhY2Nlc3MuY29tggkAnS7684Nkme0wDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQ0FAAOCAgEAJsfhsPk3r8kLXLxY+v+vHzbr4ufNtqnL9/1Uuf8NrsCtpXAoyZ0YqfbkWx3NHTZ7OE9ZRhdMP/RqHQE1p4N4Sa1nZKhTKasV6KhHDqSCt/dvEm89xWm2MVA7nyzQxVlHa9AkcBaemcXEiyT19XdpiXOP6Vhsh+J1R5m8zQOxZlV1GtF9vsXmJqWZpOVPmZ8f35BCsYPvv4yMewnrtAC8PFEK/bOPeYcKN50bol22QYaZuLfpkHfNiFTnfMh8sl/gblPyNY7DUNiP5DRcMdIwmfGQxR5WEQoHL3yPJ42Lk125zs6jIm26DGNXfwura/mi105+ENH1CaROtRYwkiHb08U6qLXXJz80mWJkT90nr8Asj35xN2cUppg74nG3YVav/38P48T56hG1NHbYFuOCske19F6wi9maUoto/3vEr0rnXJUp2KODmKdvBI7co245lHBABWikk8VfejQSlCtDBXn644ZMtAdoxKNfR2WTFVEwJiyd1Fzx0yujuiXDROLhISLQDRjhVAvawrAtLZWYK31bY7KlezPlQnl/D9Asxe85l8jO5+0LdJ6VyOs/Hd4w52alDW/MFySDZSfQHMTIc30hLBJ8OnCEIvluVQQ2UQvoW+no177N9L2Y+M9TcTA62ZyMXShHQGeh20rb4kK8f+iFX8NxtdHV3kxMEFSfDDyQ=
pia_gluetun_vm-docker  | 2022/01/06 14:44:41 INFO Shutdown successful
```

### Share your configuration

```yml
version: '3.7'
services:
  gluetun:
    image: qmcgaw/private-internet-access
    container_name: pia_gluetun_vm-docker
    cap_add:
      - NET_ADMIN
    network_mode: bridge
    ports:
      - 9888:8888/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks
      - 8001:8000/tcp # Built-in HTTP control server
# other containers ports
      - 6767:6767     # bazarr
      - 8989:8989     # sonarr
      - 7878:7878     # radarr
      - 8686:8686     # lidarr
      - 8191:8191     # flaresolverr
      - 3579:3579     # ombi
      - 4040:4040     # airsonic
      - 8096:8096     # jellyfin
      - 5850:5800     # jdownloader2
      - 9696:9696     # prowlarr
      - 8532:8532     # qbittorrent
    volumes:
      - ./data_gluetun:/gluetun
      - ./data_gluetun/port_forward:/tmp/gluetun/forwarded_port
    environment:
      - OPENVPN_IPV6=off
      - OPENVPN_ROOT=no
      - VPNSP=private internet access
      - TZ=${TIMEZONE}
      - OPENVPN_USER=${PIA_USER}
      - OPENVPN_PASSWORD=${PIA_PASS}
      - REGION=${PIA_REGION}
      - BLOCK_MALICIOUS=on
      - UPDATER_PERIOD=48h
      - PORT_FORWARDING=on
      - FIREWALL_OUTBOUND_SUBNETS=192.168.1.0/24
      - HTTPPROXY=on
      - HTTPPROXY_USER=${PROXY_USER}
      - HTTPPROXY_PASSWORD=${PROXY_PASS}
      - SHADOWSOCKS=on
      - SHADOWSOCKS_PASSWORD=${SHADOW_PASS}
      - SHADOWSOCKS_LOG=off
    restart: always
```

**Comment by hugalafutro** [comment]
as workaround revert to `v3.26.0 built on 2021-11-27T12:16:34.703Z (commit b9a9319)` using `image: qmcgaw/private-internet-access:v3.26.0`

**Comment by hiyadragon** [comment]
Having the same issue with PIA settings I reverted back to v3.26.0 👍

**Comment by qdm12** [comment]
Fixed in 6132cd9df28ed6a7ab2467c4db19cd2ceefa8f51 (building now)

Thanks for reporting!!
